aquadoggo/src/db/stores/document.rs

```diff
@@ -459,6 +459,7 @@ async fn insert_document_view(
             )
         VALUES
             ($1, $2, $3)
+        ON CONFLICT(document_view_id) DO NOTHING -- @TODO: temp fix for double document view insertions: https://github.com/p2panda/aquadoggo/issues/398
```

Comment:
Down and dirty fix for the materialization task queue duplication. (although, tbh, I don't see much wrong with solving it like this....)

Reply:
Yeah I'm also curious to hear what you think about it, what implications of such change do you see?